[14404] Added ROS 2 instructions (#23)
eProsima/vulcanexus: docs/rst/installation/linux_binary_installation.rst

Linux binary installation
=========================

> Debian packages for Vulcanexus Galactic Gamble are currently available for Ubuntu Focal. Since Vulcanexus is a ROS 2 overlay, certain ROS 2 prerequisites need to be met before installing.

Reviewer: I am not sure if Operations would like to refer to Vulcanexus as a `ROS 2 overlay`. I think the official term is that Vulcanexus is a `ROS 2 all-in-one tool set`.

ROS 2 prerequisites
-------------------

> First of all, setup a UTF-8 locale as required by ROS 2. Locale settings can be checked and set up with the following commands:

Reviewer: ```suggestion
First of all, setup a `UTF-8` locale as required by ROS 2.
Locale settings can be checked and set up with the following commands:
```

.. code-block:: bash

    locale  # check for UTF-8

    sudo apt update && sudo apt install locales
    sudo locale-gen en_US en_US.UTF-8
    sudo update-locale LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8

>     export LANG=en_US.UTF-8

Reviewer: I think we should indicate this is an example and that any other `UTF-8` locale is also valid. This one is the one used when testing ROS 2, but it is not strictly required.

ROS 2 also requires that the Ubuntu Universe repository is enabled. This can be checked and enabled with the following commands:

.. code-block:: bash

    apt-cache policy | grep universe

    # This should print something similar to:
    #
    #  500 http://us.archive.ubuntu.com/ubuntu focal/universe amd64 Packages
    # release v=20.04,o=Ubuntu,a=focal,n=focal,l=Ubuntu,c=universe,b=amd64
    #

>     # If it doesn't, run

Reviewer: ```suggestion
    # Otherwise run
```

    sudo apt install software-properties-common
    sudo add-apt-repository universe

> Now download ROS 2 GPG key into your keystore.

Reviewer: Try avoiding the direct style:

```suggestion
Now download ROS 2 GPG key into the keystore.
```

.. code-block:: bash

    sudo apt update && sudo apt install curl gnupg lsb-release
    sudo curl -sSL https://raw.githubusercontent.com/ros/rosdistro/master/ros.key -o /usr/share/keyrings/ros-archive-keyring.gpg

> And finally add ROS 2 repository to your sources list.

Reviewer: ```suggestion
And finally add ROS 2 repository to the repository manager sources list.
```

    sudo add-apt-repository ppa:beineri/opt-qt-5.15.2-focal

Next, add Vulcanexus GPG key so apt can retrieve the packages:

> 

Reviewer: I suggest keeping this blank line to separate the code block from the sentence and improve readability.
